decidim-sortitions/spec/shared/manage_sortitions_examples.rb

```diff
@@ -57,6 +57,9 @@
       let(:sortition_target_items) { ::Faker::Number.between(from: 1, to: 10) }
       let!(:proposal) { create :proposal, component: proposal_component }
 
+      it_behaves_like "having a rich text editor for field", ".tabs-content[data-tabs-content='sortition-additional_info-tabs']", "content"
```

Comment:
The sortitions specs are currently failing because this should be `full`:
```suggestion
      it_behaves_like "having a rich text editor for field", ".tabs-content[data-tabs-content='sortition-additional_info-tabs']", "full"
```